api/src/test/java/org/pmiops/workbench/api/ClusterControllerTest.java

```diff
@@ -418,6 +424,31 @@ public void testLocalize_noNotebooks() throws Exception {
     assertThat(resp.getClusterLocalDirectory()).isEqualTo("workspaces/wsid");
   }
 
+  @Test(expected = ForbiddenException.class)
+  public void listCluster_requireActiveBilling() {
+    testWorkspace.setBillingStatus(BillingStatus.INACTIVE);
```

Comment:
I'm confused why this test case needs you to both set inactive here and then stub the call below.